charts/logzio-otel-spm/README.md

```diff
@@ -0,0 +1,118 @@
+
+# Logzio otel spm
+
+##  Overview
+
+You can use a Helm chart to ship Traces to Logz.io via the OpenTelemetry collector.
+The Helm tool is used to manage packages of pre-configured Kubernetes resources that use charts.
+
+**logzio-otel-spm** allows you to ship traces and span metrics from your Kubernetes cluster to Logz.io with the OpenTelemetry collector.
+**Note:** This chart is a fork of the [opentelemtry-collector](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-collector) Helm chart.
+
+**This project is currently in beta and subject to breaking changes**
+
+**NOTE: The monitor tab in logz.io tracing calculates statistics only for `server` span kind at the moment**
+
+### Before you start you will need:
+* Helm installed on your host
+* Logz.io tracing account
+* Logz.io span metrics account. **IMPORTANT: The account name should include your tracing account name, for example if your tracing account name is `tracing`, your metrics account should be named something like `tracing-metrics`**
+
+#### Standard configuration
+
+##### Deploy the Helm chart
+
+Add `logzio-helm` repo as follows:
+
+```shell
+helm repo add logzio-helm https://logzio.github.io/logzio-helm
+helm repo update
+```
+
+##### Configure the parameters in the code
+
+Replace the Logz-io `<<tracing-token>>` with the [token](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping?product=tracing) of the tracing account to which you want to send your data.
+
+Replace the Logz-io `<<metrics-token>>` with the [token](https://app.logz.io/#/dashboard/settings/manage-tokens/data-shipping?product=metrics) of the metrics account to which you want to send your data.
+
+Replace `<<logzio-region>>` with the name of your Logz.io region e.g `us`,`eu`.
+
+##### Run the Helm deployment code
+```
+helm install  \
+--set logzio.region=<<logzio-region>> \
+--set logzio.tracing_token=<<tracing-token>> \
+--set logzio.metrics_token=<<metrics-token>> \
+logzio-otel-traces logzio-helm/logzio-otel-traces
+```
+
+##### Check Logz.io for your traces
+
+Give your traces and metrics some time to get from your system to ours, then open [Logz.io](https://app.logz.io/).
+
+### Example usage
+
+* Go to `hotrod.yml` file inside this directory.
```

Comment:
Why using otel-cluster-ip and not service name 
like here:
https://docs.logz.io/shipping/tracing-sources/dotnet-framework-otel-auto.html#define-the-logzio-otel-traces-service-name